CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 Bugfixes:
 
+* Faq: Fixed frontend JS which did not found if "yes" is checked.
```

Comment:
this will not be included in 4.0.1 anymore

it's in fact easier when you don't add a changelog, since this gives merge conflicts all the time (and I add everything in here when releasing)